research/RSOC.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<p>To chyba aż do Ok, Ants na nasze potrzeby możemy wywalić i zostawić, zostawić jedynie kilka słów typu</p><p> </p><p>Używamy frameworku [Mesa](https://mesa.readthedocs.io/en/master/tutorials/intro_tutorial.html)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SocSIM/SocSIM/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>